posts/inside-rust/2019-12-17-bisecting-rust-compiler.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "Bisecting Rust Compiler Regressions with cargo-bisect-rustc"
+author: Santiago Pastorino
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Let's say that you've just updated the Rust compiler version and have
+tried to compile your application and see a failure that wasn't there
+before. That's likely due to a regression in the compiler.  We've just
+released
+[`cargo-bisect-rustc`](https://github.com/rust-lang/cargo-bisect-rustc),
+this tool makes it super easy to find exactly when behavior has
+regressed in rustc.
```

Comment:
"We've just released `cargo-bisect-rustc`, a tool that makes it super easy to find exactly when the regression happened."